cdap-app-fabric/pom.xml

```diff
@@ -149,6 +149,10 @@
       <artifactId>MockFtpServer</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.hive</groupId>
+      <artifactId>hive-service</artifactId>
```

Comment:
I thought `HiveTokenUtils` was written in such a way as to not have any static dependency on hive jars. From looking at the code it looks like only reading string `HiveAuthFactory.HS2_CLIENT_TOKEN` needs hive jar. If we can read that string using reflection, then we can eliminate this dependency for app-fabric.